conf.d/atuin_loader.fish

```diff
@@ -4,6 +4,6 @@ functions --query \
 
 ### Set variables on load ###
 # whether or not atuin has been loaded
-_atuin_loader_load false
+set -gx _atuin_loader_load false
 # arguments to pass to `atuin init fish`
-_atuin_loader_arguments ""
+set -gx _atuin_loader_arguments ""
```

Comment:
Did you try with just the `--global` flag? I've use that [in some of my other plugins](https://github.com/dudeofawesome/fish-plugin-node-version/blob/master/conf.d/node-version.fish#L2).

Reply:
Updated to global, also works